packages/cli/src/rpc.ts

```diff
@@ -183,7 +183,9 @@ export function createProviderProxy(provider: ethers.providers.JsonRpcProvider):
     });
 
     server.on('listening', () => {
-      const addrInfo = server.address() as { address: string; family: 'IPv4' | 'IPv6'; port: number };
+      const address = server.address();
+      const addrInfo = address as { address: string; family: 'IPv4' | 'IPv6'; port: number };
```

Comment:
why separate the retyping into a separate var?

Reply:
Was using the var to console.log and seemingly did not remove it.